sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -263,8 +263,14 @@ class SQLContext(@transient val sparkContext: SparkContext)
    *
    * @group userf
    */
-  def parquetFile(path: String): DataFrame =
-    DataFrame(this, parquet.ParquetRelation(path, Some(sparkContext.hadoopConfiguration), this))
+  @scala.annotation.varargs
+  def parquetFile(path: String, paths: String*): DataFrame =
```

Comment:
This is per @rxin's [this comment] [1], which I think makes sense.

[1]: https://github.com/apache/spark/pull/4308#discussion_r23983783